mongodb.md

```diff
@@ -76,10 +75,10 @@ Within this file, you can add a `mongodb` connection using the `mongodb` driver:
 
 Once your configuration is complete, you can use the `mongodb` connection in your application:
 
-- [Using Eloquent](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/), models can be stored in MongoDB collections. In addition to the standard Eloquent features, the Laravel MongoDB package provides additional features such as embedded relationships. Using raw SQL queries is not supported obviously, as MongoDB is a NoSQL database.
+- [Using Eloquent](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/), models can be stored in MongoDB collections. In addition to the standard Eloquent features, the Laravel MongoDB package provides additional features such as embedded relationships. Raw SQL queries is not supported but advanced MongoDB queries can be used.
 - [Write complex queries](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/query-builder/) using the query builder.
-- [Using the Cache](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/cache/), the `mongodb` cache driver is optimized to use MongoDB features such as TTL indexes to automatically clear expired cache entries.
-- Using the Session with the `cache` session driver.
+- [Using the Cache](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/cache/), the `mongodb` Cache driver is optimized to use MongoDB features such as TTL indexes to automatically clear expired cache entries.
```

Comment:
Not sure why "Cache" is capitalized here, but lowercase in the subsequent bullet.

Decide if you're referring to a Laravel component by name (capitalized proper noun in that case) or not. I think in this context, lowercase makes more sense, as in the bullet below.

For "Using the <Component>" phrases, I read that as a component name for capitalization makes sense.